### DISCUSSION TITLE
【网站语言】blog.config.js配置中选择英文,然而任然是中文界面

### DISCUSSION THREAD

**Discussion by Tianqi-Yao** [tangly1024/NotionNext#1986] [Q&A]
**描述bug**
LANG: process.env.NEXT_PUBLIC_LANG || 'en-US'

**复现步骤**
blog.config.js中 LANG默认配置是
LANG: process.env.NEXT_PUBLIC_LANG || 'zh-CN'
改为
LANG: process.env.NEXT_PUBLIC_LANG || 'en-US'

**期望的正常结果**
希望变成英文界面

**截图**
这里的一些文字应该显示英文,但实际显示的是中文
<img width="199" alt="Screenshot 2023-09-22 at 12 00 29 PM" src="https://github.com/tangly1024/NotionNext/assets/25485117/6901efaf-d3fe-4d5a-b91e-714d7039dcd1">

**环境**

- 操作系统: macOS
- 浏览器: chrome
- NotionNext版本 4.0.15
- 主题 hexo

**补充说明**
在themes/hexo/components/MenuGroupCard.js输出 local 变量,发现一开始是英文,后面被别的代码改回了中文
<img width="617" alt="Screenshot 2023-09-22 at 11 59 14 AM" src="https://github.com/tangly1024/NotionNext/assets/25485117/a7e3042f-944e-4a7d-a5d0-35c1c6ad19e2">

**Comment by tangly1024** [comment]
blog.config.js 中配置的是默认语言，但是网站会自动切换成当前访问者的浏览器语言，后者优先级更高。

按您需求，我会提交一个补丁，屏蔽掉自动切换网站语言的功能

**Comment by buptmary** [comment]
我将Notion Config中的LANG配置项关闭，在blog.config.js中配置 LANG: 'zh-CN'
但是页面中仍然显示的是英文
**截图**
![image-20231121175007542](https://mary-aliyun-img.oss-cn-beijing.aliyuncs.com/typora/202311211750585.png)
